app/styles/form-elements/_inputs.scss

```diff
@@ -654,6 +654,7 @@
         grid-column: 2/4;
     }
     .sd-input__input-container {
+        min-width: 110px;
```

Comment:
is this for all inputs? Shouldn't it only be for time picker?

Reply:
Yes, changed it so that we apply it to this class only if it contains a time input